inventory/group_vars/generic/network.yml

```diff
@@ -4,18 +4,20 @@
 
 network_internal_interface: vlan100
 network_mgmt_interface: eno1
-network_workload_interface: eno2
+network_workload_interface: vlan101
 
 network_type: netplan
 
 network_ethernets:
   eno1:
     dhcp4: true
-  eno2: {}
 
 network_vlans:
   vlan100:
     id: 100
     link: "{{ network_mgmt_interface }}"
     addresses:
       - "192.168.16.{{ node_id }}/24"
+  vlan101:
+    id: 100
```

Comment:
This should be 101, no?

Reply:
Yes.Fixed with bad368cc5437a5f3bdf70ee8e73e60cd6bc2ae81.